src/RestControllers/AppointmentRestController.php

```diff
@@ -21,15 +21,50 @@
 class AppointmentRestController
 {
     private $appointmentService;
-
+    private const UNKNOW_FILTER_TYPE = "Unknow";
+    private const ARRAY_MODE = 2;
+    private const SUPPORTED_FILTER_TYPES = array(
+        "puuid",
+        "pc_title",
+        "pc_eventDate",
+        "pc_apptstatus",
+        "date_range"
+    );
+    
     public function __construct()
     {
         $this->appointmentService = new AppointmentService();
     }
 
-    public function getOne($eid)
+    /**
+     * Retrieves a single appointment by its UUID.
+     *
+     * @param string $euuid The UUID of the appointment to retrieve.
+     * @return mixed Returns the result of the appointment retrieval operation.
+     */
+    public function getOne(string $euuid)
+    {
+        $serviceResult = $this->appointmentService->getAppointment($euuid);
+        return RestControllerHelper::responseHandler($serviceResult, null, 200);
+    }
+
+    /**
+     * Updates an appointment by its UUID with the provided data.
+     *
+     * @param string $euuid The UUID of the appointment to update.
+     * @param array $data An array containing the updated appointment data.
+     * @return mixed Returns the result of the appointment update operation.
+     */
+    public function update(string $euuid, array $data)
```

Comment:
if is possible add the return type:
public function update(string $euuid, array $data): TypeToReturn.